content/en/blog/_posts/2024-08-21-cgroup-driver-from-cri.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: blog
+title: "Configuring Kubernetes just got easier: cgroup driver from CRI"
+date: 2024-08-06
```

Comment:
@a-mccarthy : Is this the correct date for publication? If not, please could you confirm what this needs to be?

Reply:
Per the filename, I'm guessing it should be 2024-08-21

Reply:
Per https://github.com/kubernetes/website/pull/47054#issuecomment-2253052256
```suggestion
date: 2024-08-21
```

Reply:
Still hasn't been resolved in the commit visible to me. Please can we update it to the latest date?

Reply:
/approve cancel

to highlight the problem

See https://github.com/kubernetes/website/pull/47054/files#r1720753283